crypto/x509/v3_addr.c

```diff
@@ -1289,9 +1288,6 @@ static int addr_validate_path_internal(X509_STORE_CTX *ctx,
             IPAddressFamily *fp =
                 sk_IPAddressFamily_value(x->rfc3779_addr, k);
 
-            if (!IPAddressFamily_check_len(fc) || !IPAddressFamily_check_len(fp))
-                return 0;
-
             if (fp == NULL) {
```

Comment:
fp = NULL check must be done first

Reply:
it was doing a check for fp being NULL after it accesses the pointer when using IPAddressFamily_check_len(fp).
If fp was NULL this would segfault.